docs/configure-artifact-repository.md

```diff
@@ -75,12 +75,15 @@ artifacts:
 
 ## Configuring AWS S3
 
-Create your bucket and access keys for the bucket. AWS access keys have the same
-permissions as the user they are associated with. In particular, you cannot
-create access keys with reduced scope. If you want to limit the permissions for
-an access key, you will need to create a user with just the permissions you want
-to associate with the access key. Otherwise, you can just create an access key
-using your existing user account.
+First, create your bucket:
+
+```bash
+export mybucket=bucket249
+aws s3 mb s3://$mybucket [--region xxx]
+```
+
+Next, create a policy which will be saved locally.
+You will attach this to the relevant authentication method in one of the sections below.
 
 ```bash
 $ export mybucket=bucket249
```

Comment:
```suggestion
```

`mybucket` is defined above now, so this is no longer needed.

also could you remove the `$ ` in the line below as well?